lib/gems/pending/appliance_console/database_replication_standby.rb

```diff
@@ -54,14 +70,16 @@ def ask_for_repmgrd_configuration
 
     def activate
       say("Configuring Replication Standby Server...")
-      data_dir_empty? &&
+      initialize_postgresql_disk if disk
+      PostgresAdmin.prep_data_directory &&
+        data_dir_empty?
```

Comment:
Not sure I understand fully how you are thinking about this. How about:
1. initialize_postgresql_disk if disk (initialize the disk if the user selected one)
2. PostgresAdmin.prep_data_directory (prepare data dir. here we remove '&&' )
3. check data_dir_empty?

or should it be the other way around, so check data_dir_empty first and then move to the next?
I am afraid in this case we might also fail if a user tried to configure the internal database before (so we have a valid pg dir, despite the content being disposable).

what do you think?